lib/eunit/src/eunit.erl

```diff
@@ -140,6 +140,9 @@ test(Tests) ->
 %% not automatically execute tests found in related module suffixed with "_tests".
 %% This behaviour might be unwanted if execution of modules found in a folder
 %% is ordered while it contains both source and test modules.</dd>
+%% <dt>`scale_timeouts'</dt>
+%% <dd>If this numeric value is set, timeouts will get scaled accordingly.
+%% It may be useful when running a set of tests on a slower host.</dd>
```

Comment:
maybe you could add a simple example, since this option is new?
Something along these lines would be useful, I think
https://github.com/erlang/otp/pull/7635/files#diff-6c0970985d8fa6ff610aeaf3ec2250c40bd9982db4f26f72b9e67eccc52b9204L129-R130

Reply:
Absolutely, I updated the description with a couple of examples.